pages.ru/common/echo.md

```diff
@@ -0,0 +1,24 @@
+# echo
+
+> Отобразить заданные аргументы.
+> Больше информации: <https://www.gnu.org/software/coreutils/echo>.
+
+- Отобразить текстовое сообщение. Примечание: кавычки необязательны:
+
+`echo "{{Привет, мир}}"`
+
+- Отобразить сообщение с переменной окружения:
+
+`echo "{{Мой путь - $PATH}}"`
+
+- Отобразить сообщение не перенося каретку на новую строку:
+
+`echo -n "{{Hello World}}"`
```

Comment:
```suggestion
- Отобразить сообщение не перенося каретку на новую строку:

`echo -n "{{Привет, мир}}"`
```

Optional fix: broken consistency: translated placeholder content expected.

Reply:
@EmilySeville7cfg thank you, 
I found another mistake - the lack of a comma.

```suggestion
- Отобразить сообщение, не перенося каретку на новую строку:

`echo -n "{{Привет, мир}}"`
```

Reply:
Thanks :)